config.staging.yaml

```diff
@@ -16,4 +16,4 @@ metadata:
   name: global-config
   namespace: staging
 data:
-  environment: "staging"
+  ENVIRONMENT: "staging"
```

Comment:
Why make this upper-case? I think that's confusing, 'cos it implies it's an environment variable itself, which it isn't. It's just a key in the configMap, which *may* then be used inside an environment variable, e.g. with:

``` yaml
    configMapKeyRef:
        name: global-config
        key: environment
```